src/Nethermind/Nethermind.Consensus/Processing/CensorshipDetector/CensorshipDetector.cs

```diff
@@ -61,8 +67,18 @@ public CensorshipDetector(
         _blockProcessor.BlockProcessing += OnBlockProcessing;
     }
 
+    private bool IsSyncing()
+    {
+        long bestSuggestedNumber = _blockTree.FindBestSuggestedHeader()?.Number ?? 0;
+        long headNumberOrZero = _blockTree.Head?.Number ?? 0;
+        return bestSuggestedNumber > headNumberOrZero + 8;
```

Comment:
we probably shouldn't even do `8` threshold, as here we should only check tx pool if we are processing current head block, right?

Reply:
Right, my bad